LabManual.md

```diff
@@ -244,7 +244,7 @@ Click **Search** to search for all documents.
 
 We will be developing a bot using the C# SDK.  To get started, you need two things:
 1. The Bot Framework project template, which you can download from http://aka.ms/bf-bc-vstemplate.  The file is called "Bot Application.zip" and you should save it into the &lt;Documents&gt;\Visual Studio 2015\Templates\ProjectTemplates\Visual C#\ directory (_note: if you're on VS2017, adjust the path accordingly_).  Just drop the whole zipped file in there; no need to unzip.  
-2. Download the Bot Framework Emulator for testing your bot locally from https://emulator.botframework.com/.  
+2. Download the Bot Framework Emulator for testing your bot locally from https://emulator.botframework.com/.  The emulator installs to c:\Users\your-username\AppData\Local\botframework\app-3.5.27\botframework-emulator.exe.  
```

Comment:
Consider `c:\Users\`_your-username_`\AppData\Local\botframework\app-3.5.27\botframework-emulator.exe` to make it obvious which parts they need to sub.